mix.exs

```diff
@@ -1,13 +1,13 @@
 defmodule GenRMQ.Mixfile do
   use Mix.Project
 
-  @version "1.3.0"
+  @version "2.0.0"
```

Comment:
Do we also need to update the README?

Reply:
@vorce It is here https://github.com/meltwater/gen_rmq/pull/103/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8. Are you thinking about smth more?

Reply:
@mkorszun Ah ok, no that is what I meant 👍